Meadowlark-js/packages/meadowlark-core/src/middleware/LogAnonymizedRequestMiddleware.ts

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import { isDebugEnabled, Logger } from '@edfi/meadowlark-utilities';
+import { writeRequestToLog } from '../Logger';
+import { MiddlewareModel } from './MiddlewareModel';
+
+const moduleName = 'core.middleware.ParseBodyMiddleware';
+
+export async function anonymizeAndLogRequestBody({
+  frontendRequest,
+  frontendResponse,
+}: MiddlewareModel): Promise<MiddlewareModel> {
+  // if there is a response already posted, we are done
+  if (frontendResponse != null) return { frontendRequest, frontendResponse };
```

Comment:
How would we ever fall through to line 19?

Reply:
The frontend response is generally null at this point, if there is already a response we don't continue processing

Reply:
I got myself mixed up between request and response. This makes sense to me now, when I read it correctly.